Week-3/exercise_4/index.js

```diff
@@ -0,0 +1,31 @@
+/*
+when "createStack" function returns "push" and "pop" inner function, then function bundled with its lexical scope and creates a closure. That is why outside of "createStack" function  cannot access "items" variable, but it can access only in the function which is inside of the "createStack"
+
+*/
+
+function createStack() {
+    // Write your code here...
+    const items = [];
+    return (function () {
```

Comment:
Do we really need to use IIFE ?

Reply:
IIFE helps in a way to execute functions immediately, as soon as they are created. IIFEs are very useful because they don't pollute the global object, and they are a simple way to isolate variable declarations. **But** I pushed another method too so, please let me know if this one is correct or not ?